File: c++/CFLOW007.cc
#include <iostream>
using namespace std;

int main()
{
    int z;
    cin >> z;
    int result = 0;
    while (z > 0)
    {
        int a;
        cin >> a;
        while (a > 0)
        {
            int b = a % 10;
            result = (result * 10) + b;
            a /= 10;
        }
        cout << result << endl;
        result = 0;
        z -= 1;
    }
}

File: scala/CFLOW007.scala
object CFLOW007 {
  def main(args: Array[String]): Unit = {
    var z = scala.io.StdIn.readInt();
    var result = 0
    while (z > 0) {
      var a = scala.io.StdIn.readInt();
      while (a > 0) {
        var b = a % 10
        result = (result * 10) + b
        a /= 10
      }
      println(result)
      result = 0
      z -= 1
    }
  }
}